src/presets.js

```diff
@@ -119,4 +119,9 @@ module.exports = {
       showNotFound: true,
     },
   },
+  babel: {
+    System: ['OS'],
+    Binaries: ['Node', 'npm', 'Yarn'],
+    npmPackages: '{*babel*,@babel/*,eslint,webpack,create-react-app,react-native,lerna}',
```

Comment:
Maybe also `jest`? In the past we got different issues about incompatible jest versions.